### PR TITLE
Add docs about using elasticsearch as a storage driver

### PR DESCRIPTION
**Pull request by nik-kor**
It's strange that there is not docs about ES storage driver. It would be useful for me.

### CODE REVIEW

File: docs/elasticsearch.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Exporting cAdvisor Stats to Elasticsearch
```
Comment (timstclair) [inline docs/elasticsearch.md]: Capitalize the "search" in ElasticSearch, same below.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ cAdvisor supports exporting stats to [InfluxDB](https://influxdb.com/). See the
 
 cAdvisor also exposes container stats as [Prometheus](http://prometheus.io) metrics. See the [documentation](docs/prometheus.md) for more information.
 
+cAdvisor also supports exporting stats to [Elasticsearch](https://www.elastic.co/). See the [documentation](docs/elasticsearch.md) for more information.
```
Comment (timstclair) [inline README.md]: nit: Capitalize the "search" in ElasticSearch

File: docs/elasticsearch.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+```
+ -storage_driver=elasticsearch
+```
+
+Specify ES host address.
+
+```
+ -storage_driver_es_host="http://elasticsearch:9200"
+```
+
```
Comment (timstclair) [inline docs/elasticsearch.md]: Can you add a comment about the optional flags too:
```
	argTypeName      = flag.String("storage_driver_es_type", "stats", "ElasticSearch type name")
	argEnableSniffer = flag.Bool("storage_driver_es_enable_sniffer", false, "ElasticSearch uses a sniffing process to find all nodes of your cluster by default, automatically")
```
(The comment on the enable_sniiffer flag is confusing, since it's disabled by default)

File: docs/elasticsearch.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+# Examples
+
+[very detailed tutorial here](https://github.com/gregbkr/docker-elk-cadvisor-dashboards)
```
Comment (timstclair) [inline docs/elasticsearch.md]: nit: make this a full sentence, e.g.
> For a detailed tutorial, see [docker-elk-cadvisor-dashboards](...)

**Comment by timstclair** [comment]
Thanks! I've been meaning to separate the storage documentation from the main readme, but this is ok for now.

**Comment by timstclair** [comment]
ok to test

**Comment by nik-kor** [comment]
I signed it!

File: docs/elasticsearch.md (hunk 3 above)
Comment (nik-kor) [inline docs/elasticsearch.md]: done

File: docs/elasticsearch.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+```
+ -storage_driver_es_host="http://elasticsearch:9200"
+```
+
+There are also optional flags:
+
+```
+ # ElasticSearch type name. By default it's "stats".
+ -storage_driver_es_type="stats"
+ # ElasticSearch uses a sniffing process to find all nodes of your cluster by default, automatically. False by default.
```
Comment (timstclair) [inline docs/elasticsearch.md]: Can you reword this to something like, `ElasticSearch can use a sniffing process to find all nodes of your cluster automatically. False by default.`

**Comment by timstclair** [comment]
One last comment, then LGTM.

**Comment by nik-kor** [comment]
done. rebased it

**Comment by timstclair** [comment]
Thanks! LGTM
